src/Detail/Blitline/Job/Definition/JobDefinition.php

```diff
@@ -2,62 +2,95 @@
 
 namespace Detail\Blitline\Job\Definition;
 
-use Detail\Blitline\Job\Source;
-
 class JobDefinition extends BaseDefinition implements JobDefinitionInterface
 {
     const OPTION_SOURCE       = 'src';
     const OPTION_POSTBACK_URL = 'postback_url';
     const OPTION_VERSION      = 'v';
     const OPTION_FUNCTIONS    = 'functions';
+    const OPTION_SOURCE_TYPE  = 'src_type';
+    const OPTION_SOURCE_DATA  = 'src_data';
 
     /**
      * @var array
      */
     protected $options = array(
-        self::OPTION_VERSION   => '1.21',
+        self::OPTION_VERSION   => '1.22',
```

Comment:
Is the feedback I've received from Blitline support for the PDF processing "https://www.blitline.com/v3/home/gist?gist_id=7bcc632672e38bcf5e2a4d01899ca9cd"

I've not tried if it works **only** with version 1.22